code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/notification/models/CarouselPushTemplate.kt

```diff
@@ -47,7 +48,8 @@ internal open class CarouselPushTemplate : AEPPushTemplate {
         val interactionUri: String?
     )
 
-    constructor(data: Map<String, String>) : super(data) {
+    @Throws(NotificationConstructionFailedException::class)
```

Comment:
I am not seeing this exception thrown from the constructor.

Reply:
Removed